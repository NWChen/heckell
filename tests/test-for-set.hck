for s in {1, 100, 500} do
	print(s);
end