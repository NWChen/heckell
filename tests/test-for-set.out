1
100
500
